pkg/cover/report.go

```diff
@@ -135,7 +135,7 @@ func (rg *ReportGenerator) prepareFileMap(progs []Prog, debug bool) (fileMap, er
 	}
 	// If the backend provided coverage callback locations for the binaries, use them to
 	// verify data returned by kcov.
-	if len(unmatchedProgPCs) > 0 {
+	if (len(unmatchedProgPCs) > 0) && !force {
```

Comment:
Please remove (), Go does not do insane things with operator priorities like C/C++.  Parentheses rarely needed in Go.